test/parallel/test-abort-uncaught-exception.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const spawn = require('child_process').spawn;
+const node = process.execPath;
+
+if (process.argv[2] === 'child') {
+  setImmediate(function() {
+    throw new Error('child error');
+  });
+} else {
+  run('', null);
+  run('--abort-on-uncaught-exception', 'SIGABRT');
+}
+
+function run(flags, signal) {
+  let args = [__filename, 'child'];
+  if (flags)
+    args.unshift(flags);
+
+  const child = spawn(node, args);
+  child.on('exit', function(code, sig) {
```

Comment:
It's better to have `common.mustCall` here. If the `exit` is not emitted because of any other bug then this test will still pass.